downloaded/installLisk.sh

```diff
@@ -102,6 +102,18 @@ user_prompts() {
 }
 
 download_lisk() {
+	if [[ "$LOCAL_TAR" ]]; then
+		echo -e "\\nUsing local binary $LOCAL_TAR"
+		LISK_VERSION="$LOCAL_TAR"
+		readarray LISK_DIR_A < <(tar -tf "$LISK_VERSION" | cut -d '/' -f 1 | sort -u | tr -d '\n')
+		if [[ ${#LISK_DIR_A[@]} != 1 ]]; then
+			echo -e "\\nMalformed binary, aborting..."
```

Comment:
Our tarballs always contain a single top-level directory called the same as the tarball AFAIK (i.e. `lisk-x.y.z-Linux-x86_64.tar.gz` contains one (and only one) top-level directory named `lisk-x.y.z-Linux-x86_64`)

Reply:
Ours do, but if someone's rolling their own.....but in that case, they're probably on their own anyway. I'll simplify this